assets/javascripts/discourse/components/group-tracker-first-post.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if @topic.first_tracked_post}}
+  <DButton
+    class="first-tracked-post"
+    @icon="arrow-circle-up"
+    @title="group_tracker.first_post"
+    @disabled={{this.disabled}}
+           @action={{action "jumpToFirstTrackedPost"}}
```

Comment:
Can we fix the indentation here?

Reply:
Done - looks like my editor needs a prettier update, too.